docs/modules/language-reference/pages/index.adoc

```diff
@@ -3557,6 +3555,20 @@ res4 = Set(1, "x", 5.min, List(1, 2, 3)) // <4>
 <3> result: same set of length 3
 <4> result: heterogeneous set that contains a list as its last element
 
+Sets retain the order of elements when constructed, which impacts how they are iterated over.
+However, this order is not considered when determining equality of two sets.
```

Comment:
```suggestion
However, this order is not considered when determining equality or hashing of two sets.
```

Reply:
Hashing is only an implementation detail; it's not part of the language.

Reply:
Fair, though it may be worth explicitly adding that differently ordered sets are also equivalent map/mapping keys.

Reply:
That feels oddly specific? I think the equality bit also implies that they are treated as equivalent mapping keys.